src/SDKs/CognitiveServices/dataPlane/Vision/Microsoft.CognitiveServices.Vision/Microsoft.CognitiveServices.Vision.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.CognitiveServices.Vision</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services Vision APIs.</Description>
+    <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
@DavidLiCIG please mark this version as preview if this is a preview release. Release notes say this is preview but the version is not marked accordingly.
Developers who will be downloading from nuget will be confused what is the case, is it preview or not.